### ISSUE TITLE
Binary releases wen?

### ISSUE THREAD

**Issue by sigmap** [caitp/TwitchSwitcher#21]
When should we expect any sort of binary releases to appear on the repo?

**Comment by caitp** [comment]
Yeah, that would be a good thing to have. I don't really have anywhere to host a binary release, at the moment, though.

**Comment by BrownZFilmZ** [comment]
MEGA, Drive, and I mean Here?

**Comment by sigmap** [comment]
And sourceforge as well.

**Comment by caitp** [comment]
Yeah, there are lots of options, a lot of them are not particularly good choices, but maybe github's hosting will suffice. Maybe this weekend I'll update it to be compatible with the latest libobs and host a binary.

Keep in mind, this is definitely pre-release quality stuff, I haven't been spending much time on it since I haven't been streaming much this year :)

**Comment by ChristopheCVB** [comment]
That's a lot of weekends since your reply ;) Any chance getting the binaries soon ?

**Comment by Bluscream** [comment]
YES PLEASE NOW YET! ALREADY!!!

**Comment by caitp** [comment]
I AM SORRY

If you have experience with C++ & CMake, Please test this out and see if it's working with recent OBS releases. I simply have not had the time and probably won't for a while.

**Comment by Bluscream** [comment]
I can't :c
